src/include/sof/audio/component.h

```diff
@@ -545,7 +545,7 @@ static inline struct comp_dev *comp_alloc(const struct comp_driver *drv,
 {
 	struct comp_dev *dev = NULL;
 
-	dev = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM, bytes);
+	dev = rzalloc(SOF_MEM_ZONE_RUNTIME_SHARED, 0, SOF_MEM_CAPS_RAM, bytes);
```

Comment:
every component should belong to only one pipeline and scheduled on its specific DSP core only, this is not needed?

Reply:
@keyonjie these twp are are needed today to get dynamic pipelines to work with multi-core. pipelines have info about sched comp's that could be on a separate core. 

This PR is not intended to go in as in. The idea to start with a stable base were everything works and then revert back to cached one by one